### PR TITLE
Add entries for Container Registry Client

### PR DESCRIPTION
**Pull request by jeremymeng**
- add package entry
- update TOC mapping

### CODE REVIEW

File: ci-configs/packages-latest.json [hunk 1]
```diff
@@ -563,6 +563,9 @@
     {
       "name": "@azure/communication-sms"
     },
+    {
```
Comment (scbedd) [inline ci-configs/packages-latest.json:566]: This is a preview package yes? It's already been added [over here](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/9fa7add4f28c553149f2c6535194a62dc912a042/ci-configs/packages-preview.json#L133).

File: docs-ref-mapping/reference-unified.yml [hunk 2]
```diff
@@ -426,6 +426,12 @@
       uid: azure.nodejs.sdk.landingpage.services.containerregistry
       href: ~/docs-ref-services/{moniker}/container-registry.md
       items:
+      - name: Client
+        uid: azure.nodejs.sdk.landingPage.services.containerregistry.Client
+        href: ~/docs-ref-services/{moniker}/container-registry-readme.md
```
Comment (scbedd) [inline docs-ref-mapping/reference-unified.yml:431]: Looks great!

**Review by scbedd — CHANGES_REQUESTED** [review]
Remove changes to package-latest. G2g otherwise.

File: ci-configs/packages-latest.json (hunk 1 above)
Comment (jeremymeng) [reply ci-configs/packages-latest.json:566]: Yes. I didn't know there's another file for preview packages.

Comment (jeremymeng) [reply ci-configs/packages-latest.json]: Fixed!

File: docs-ref-mapping/reference-unified.yml [hunk 3]
```diff
@@ -426,6 +426,12 @@
...
+        landingPageType: Service
```
Comment (scbedd) [inline docs-ref-mapping/reference-unified.yml:432]:
```suggestion
```

**Review by scbedd — APPROVED** [review]
@jeremymeng please accept my suggestion before merging. You only need `landingPageType: Service` if you don't have a `href`. Which you have set.

**Pull request merged**